tests/tools/oecert/README.md

```diff
@@ -23,6 +23,19 @@ where Options are:
     --log LOG_FILENAME    : log file name, default: oecert.log
     --verbose             : dump verbose info of evidence
 
+Example 1 Generate, verify and dump a ceritificate:
```

Comment:
append "to default output file."

Same below

Reply:
Done